src/module_PI/Raspberry_PI/main/PacketMessageTranslator.java

```diff
@@ -0,0 +1,110 @@
+package module_PI.Raspberry_PI.main;
+
+import java.util.*;
+
+public class PacketMessageTranslator {
+	
+	private Map<Integer, String> messageTable = new Hashtable<>();
+	private final int MAX_PACKET_DETAILS_INT = 55999;
+	private final int NUM_OF_BYTES_IN_PACKET_DETAILS = 3;
+	private final int NUM_OF_BYTES_IN_PACKET = 5;
+	private final int MAX_DETAILS_ONLY_INT = 999;
+	private final int MIN_FULL_PACKET_INT = 11001;
+	
+	public PacketMessageTranslator(){
+		buildTable();
+	}
+	
+	private void buildTable(){
+		messageTable.put(1, "Execute Command: Arm");
```

Comment:
Sooo honestly I want to move on to the logging system for now. These tests work, I think it would be useful for me to read that book first as soon as I finish this book on Git. After we turn it in then maybe I can look at improving these tests.